res/values/strings.xml

```diff
@@ -922,16 +920,14 @@
     <string name="secure_join_started">%1$s invited you to join this group.\n\nWaiting for the device of %2$s to reply…</string>
     <!-- placeholder will be replaced by the name of the inviter. -->
     <string name="secure_join_replies">%1$s replied, waiting for being added to the group…</string>
-    <string name="contact_verified">%1$s verified.</string>
-    <string name="contact_not_verified">Cannot verify %1$s.</string>
-    <!-- Shown in contact profile. The placeholder will be replaced by the name of the contact that verified the contact shown. -->
-    <string name="verified_by">Verified by %1$s</string>
-    <string name="verified_by_you">Verified by me</string>
+    <string name="contact_verified">%1$s introduced.</string>
+    <string name="contact_not_verified">Cannot introduce %1$s.</string>
```

Comment:
this gets a bit confusing, now it is not about verification but "introduction" from an user point of view the contact is "introduced" immediately after scanning and the chat appears, so "cannot introduce X" doesn't tell them anything since they don't know what was going on in the background

Reply:
suggestions welcome :)

Reply:
maybe we can pick up the e2ee term here: `Cannot establish guaranteed end-to-end encryption with %1$s`

Reply:
i think, i will just change the existing string, so that ui do not need to updates. translators get aware of the change somehow, at least this was the case in the past